android/app/build.gradle

```diff
@@ -68,7 +68,7 @@ flutter {
 
 
 dependencies {
-    implementation project(':unityLibrary')
+   // implementation project(':unityLibrary')
```

Comment:
Remover '//'. Essa linha é necessária na master. Manter descomentado quando for para ela.